feat: render coordinates

cymruu opened the pull request:
demo: 
```http://127.0.0.1:6175/image.gif?fen=4k3/6KP/8/8/8/8/7p/8&orientation=black&lastMove=a1a2```
![image](https://user-images.githubusercontent.com/2981598/208441855-5ce812fc-d069-4497-a76b-69227706ffbd.png)

```http://127.0.0.1:6175/image.gif?fen=4k3/6KP/8/8/8/8/7p/8&lastMove=h1h2```
![image](https://user-images.githubusercontent.com/2981598/208441931-3bd9f5f9-4643-416a-8cbd-3ceb47e21185.png)

```http://127.0.0.1:6175/example.gif```
https://github.com/cymruu/lila-gif/blob/cymruu/render-coordinates/example.gif?raw=true

closes #128

niklasf commented on src/api.rs:
```diff
@@ -102,6 +102,58 @@ impl<'de> Deserialize<'de> for CheckSquare {
+            {
+                if name == "1" || name == "yes" || name == "true" {
+                    Ok(Coordinates::Yes)
+                } else {
```
Let's report an error if the parameter is not a known falsy name.

niklasf reviewed (changes requested):
Thanks! This looks pretty good, already.

On Lichess, the coordinates are rendered in the opposite square color, e.g., light brown on dark brown and vice versa. Maybe that would be a bit more legible?

cymruu commented:
@niklasf Thanks for reviewing. I have addressed your comments. Here are the results:
![image](https://user-images.githubusercontent.com/2981598/209120442-dcb05ac9-fa39-4635-a073-72d6d6b3fcb1.png)

![image](https://user-images.githubusercontent.com/2981598/209120526-506224ac-76b6-4dd3-9534-beaacb8e16df.png)

https://github.com/lichess-org/lila-gif/raw/7de92acac80dfc74377866bc2ebf5857a5124927/example.gif

niklasf commented:
Looks good! Deployed to prod.
